packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -34,11 +34,13 @@ export interface FullWidthAlertProps {
   description?: React.ReactNode
   primaryButtonConfig?: AlertButtonConfig
   secondaryButtonConfig?: AlertButtonConfig
+  tertiaryButtonConfig?: AlertButtonConfig
```

Comment:
If anyone needs a 4th button that person has to refactor this into an array 😉

Reply:
I don't think the current layout will allow for a 4th button today